src/ParachainGovernance.sol

```diff
@@ -151,29 +149,26 @@ contract ParachainGovernance is Parachain {
         _thisVote.initiator = _disputeInitiator;
         _thisVote.blockNumber = block.number;
         _thisVote.startDate = block.timestamp; // This is correct bc consumer parachain must submit votes before voting period ends
-        _thisVote.voteRound = voteRounds[_disputeId].length;
+        _thisVote.voteRound = voteRounds[_disputeId];
 
         disputeIdsByReporter[_disputedReporter].push(_disputeId);
 
-        if (voteRounds[_disputeId].length == 1) {
-            // Assumes voteRounds[_disputeId] will never be empty
+        if (voteRounds[_disputeId] == 1) {
             require(block.timestamp - _timestamp < 12 hours, "Dispute must be started within reporting lock time");
             // slash a single stakeAmount from reporter
             _thisDispute.slashedAmount = parachainStaking.slashParachainReporter(
                 _slashAmount, parachain.id, _thisDispute.disputedReporter, address(this)
             );
             _thisDispute.value = _value;
```

Comment:
We can just save all of the dispute info on round 1, (thisDispute.paraId, thisDispute.queryId, etc.)

Reply:
Good idea. Done